tests/TestSuite/Mutant/ResultTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+*
+* @category   Humbug
+* @package    Humbug
+* @copyright  Copyright (c) 2015 Pádraic Brady (http://blog.astrumfutura.com)
+* @license    https://github.com/padraic/humbug/blob/master/LICENSE New BSD License
+* @author     Thibaud Fabre
+*/
+namespace Humbug\Test\TestSuite\Mutant;
+
+use Humbug\TestSuite\Mutant\Result;
+
+class ResultTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetStatusCodeReturnsCorrectCode()
+    {
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(true, true, true));
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(false, false, true));
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(true, false, true));
```

Comment:
For this PR it will be OK. 
Passing boolean two function always should burn red light. That means that you have two behaviors in your method. In fact you should have two methods.
Here you have 3 booleans, so in fact you have 8 behaviors(8 separate methods). 
Each new boolean in this method multiplies original behaviors by 2. So complexity rises exponential.